notifications/broadcast.py

```diff
@@ -82,13 +82,10 @@ def __init__(
         if attachments is not None:
             self.validate_list_types(attachments)
 
-        # ipdb.set_trace()
-        print(notification_pk)
         if Notification.objects.filter(pk=notification_pk).exists():
             self.notification_pk = notification_pk
```

Comment:
self.notification_pk is not needed. it is always available as self.obj.id. btw, this obj needs to change to something more descriptive